gitlab/2_configure_oauth_app.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+
+if [[ $(kubectl get pods -n eclipse-che 2> /dev/null | wc -l) -gt 0 ]]
+then
+  NAMESPACE="eclipse-che"
+else
+  NAMESPACE="openshift-devspaces"
+fi
```

Comment:
maybe it makes sense to print a message that will ask a user to deploy che/DS in case if NAMESPACE is empty? 
To avoid `Error from server (NotFound): namespaces "openshift-devspaces" not found`

Reply:
Fixed